### PR TITLE
Fixes #7634: Make NodeInfo more dependant on Node

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/7634

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
PR updated

File: rudder-core/src/main/scala/com/normation/rudder/domain/nodes/NodeInfo.scala [hunk 1]
```diff
@@ -62,13 +60,21 @@ case class NodeInfo(
...
   //TODO: remove isPolicyServer, and pattern match on
   //      on role everywhere.
   , serverRoles   : Set[ServerRole]
-) extends HashcodeCaching
+) extends HashcodeCaching {
+
+  val id                         = node.id
+  val name                       = node.name
+  val description                = node.description
+  //val creationDate               = node.creationDate
```
Comment (fanf) [inline rudder-core/src/main/scala/com/normation/rudder/domain/nodes/NodeInfo.scala]: Why do you remove creationDate, nodeReportingConfiguration and properties which are not in Node ?

Comment (fanf) [inline rudder-core/src/main/scala/com/normation/rudder/domain/nodes/NodeInfo.scala]: OK, at least creationDate (not sure the other two are actually used)

**Comment by VinceMacBuche** [comment]
PR updated

**Pull request merged**